src/main/thrift/parquet.thrift

```diff
@@ -282,13 +282,14 @@ struct Statistics {
 }
 
 /** Empty structs to use as logical type annotations */
-struct StringType {}  // allowed for BINARY, must be encoded with UTF-8
-struct UUIDType {}    // allowed for FIXED[16], must encoded raw UUID bytes
-struct MapType {}     // see LogicalTypes.md
-struct ListType {}    // see LogicalTypes.md
-struct EnumType {}    // allowed for BINARY, must be encoded with UTF-8
-struct DateType {}    // allowed for INT32
-struct Float16Type {} // allowed for FIXED[2], must encoded raw FLOAT16 bytes
+struct StringType {}        // allowed for BINARY, must be encoded with UTF-8
+struct UUIDType {}          // allowed for FIXED[16], must encoded raw UUID bytes
+struct MapType {}           // see LogicalTypes.md
+struct ListType {}          // see LogicalTypes.md
+struct EnumType {}          // allowed for BINARY, must be encoded with UTF-8
+struct DateType {}          // allowed for INT32
+struct Float16Type {}       // allowed for FIXED[2], must encoded raw FLOAT16 bytes
+struct FixedSizeListType {} // see LogicalTypes.md
```

Comment:
Something is missing here. Shouldn't this type contain the element type? And the length of the list? The length of the list could be deduced from the size of the underlying `fixed_len_byte_array`, but at least the element type would be necessary then.